README.md

```diff
@@ -69,6 +69,7 @@ Option|Default|Type|Description
 -|-|-|-
 `--browser`|`chromium`|chromium, firefox, or webkit|Which browser to run the tests in
 `--build-dir`|`storybook-static`|string|Storybook static build directory
+`--storybook-address`||string|Storybook server address to test against instead of using a static build directory. If set, `--build-dir` will be ignored. e.g. --storybook-address `http://localhost:6006`
```

Comment:
I'm not fully sure I like the name of this flag. I don't hate it, but I think it could be clearer as to what it does. Any thoughts?